lib/ddtrace/contrib/grape/endpoint.rb

```diff
@@ -196,6 +204,17 @@ def analytics_sample_rate
             datadog_configuration[:analytics_sample_rate]
           end
 
+          def exception_is_error?(exception)
+            matcher = datadog_configuration[:error_responses]
+            return false unless exception
+            if exception.respond_to?('status') && matcher && matcher.set_range.include?(exception.status)
```

Comment:
I think the implementation looks more clear if we hide the fact the we have a `set_range` internally.

If we only expose the methods that we'd like the user to have access to, I believe the code looks more organized:

```suggestion
            if exception.respond_to?('status') && matcher && matcher.include?(exception.status)
```

This requires implementing `def include?(status)` in the StatusCodeMatcher class.

Reply:
Sounds good!

Reply:
Ditto to @marcotc comments, and also just suggesting that `&& matcher` is not necessary if we have a default